### PR TITLE
Feature: Fix Merge Relations Redundant call , Add Third Party to Company Type, Chore: update API Schema based on Swagger

### PR DESCRIPTION
**Pull request by rishirajkumar97**
### Whats Changed

- Updated API Schema based on Latest Swagger
- Added Third Party to Company Type
- Removed Merge relations in api_record which is a redundant query to Platform for Every Post /Bulk Import / Update call via gem.

### CODE REVIEW

File: app/models/lockstep/connection.rb [hunk 1]
```diff
@@ -3,7 +3,7 @@ class Lockstep::Connection < Lockstep::ApiRecord
   self.id_ref = 'company_id'
   load_schema(Schema::Company)
 
-  enum company_type: %w[Customer Vendor Group Company]
+  enum company_type: ['Customer', 'Vendor', 'Group', 'Company', 'Third Party'].freeze
```
Comment (daksh1103) [inline app/models/lockstep/connection.rb:6]: @rishirajkumar97  why don't we have a company_type CustomerVendor here ? we do see it at Inbox end rite ?

File: app/platform_api/schema/email_reply_generator_response.rb [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+  # The id for this request in the GMS system
+  # @type: string
+  # @format: uuid
+  field :message_id
+
+
+  has_many :suggestions, {:class_name=>"Schema::EmailReplyGeneratorSuggestions", :included=>true}
+
+end
```
Comment (daksh1103) [inline app/platform_api/schema/email_reply_generator_response.rb:16]: Rubocop issue , add an extra line at the end of the file

File: app/platform_api/swagger.json [hunk 3]
```diff
@@ -11348,7 +11363,7 @@
           "Payments"
         ],
         "summary": "Retrieve payment PDF",
-        "description": "Retrieves a PDF file for this payment if it has been synced using an app enrollment to one of the supported apps.\r\n\r\nQuickbooks Online supports AR Payments.\n\n### Roles\n\nTo call this endpoint, you must have one of these roles:\n\n* Group Owner\n* Group Admin\n* Member\n* Read-Only\n\n\nYou can view your roles with the [Status](https://developer.lockstep.io/reference/get_api-v1-status) API.",
+        "description": "Retrieves a PDF file for this payment if it has been synced using an app enrollment to one of the supported apps.\r\n\r\nQuickBooks Online supports AR Payments.\n\n### Roles\n\nTo call this endpoint, you must have one of these roles:\n\n* Group Owner\n* Group Admin\n* Member\n* Read-Only\n\n\nYou can view your roles with the [Status](https://developer.lockstep.io/reference/get_api-v1-status) API.",
```
Comment (daksh1103) [inline app/platform_api/swagger.json:11366]: i see a new schema being added after we generate the schema from Swagger of platform , shouldn't we update them on ReadME also ?

File: app/models/lockstep/connection.rb (hunk 1 above)
Comment (rishirajkumar97) [reply app/models/lockstep/connection.rb:6]: No we dont

File: app/platform_api/swagger.json (hunk 3 above)
Comment (rishirajkumar97) [reply app/platform_api/swagger.json:11366]: Readme is our documentation , they dont need to be updated.

**Pull request merged**